tests/src/test/scala/actionContainers/JavaActionContainerTests.scala

```diff
@@ -87,7 +87,7 @@ class JavaActionContainerTests extends FlatSpec with Matchers with WskActorSyste
         },
         env.take(1).toMap)
 
-    out.trim shouldBe empty
+    out.trim should fullyMatch regex """^exec -a 'javaAction-all.jar'(.*) -jar /deployments/javaAction-all.jar$"""
```

Comment:
Here and below: why log these?

Reply:
these logs are from base docker image - just to easily debug whats the JVM options its started with.

Reply:
I suppose that’s useful - it counts against the users log limit for an action. How long is the output I don’t have a chance to run this atm.

Reply:
its just at the start of the container and hardly max 2 lines

Reply:
Thanks.